crates/lib3h/src/engine/real_engine.rs

```diff
@@ -187,8 +174,25 @@ impl<T: Transport, D: Dht> NetworkEngine for RealEngine<T, D> {
     }
 }
 
+/// Drop
```

Comment:
Do we need tests for the `Drop` traits or is this tested naturally by virtue of any transport test successfully finishing?

Reply:
yes, exactly. You'd see that warn log in the tests logs if something errored.